Leave halcs-fe services running when AFC BPM board is removed

guilhermerc opened the pull request.

guilhermerc committed:
stop-fpga-program.sh: leave halcs-fe services running

Restarting halcs-fe services sometimes locks up halcs communication with
RFFE communication and one have to physically hard-reset it. So, keep
halcs-fe services running even if its corresponding BPMs' PCIe device is
removed.

guilhermerc commented:
@ericonr , can we merge this?

ericonr commented on src/apps/halcsd/init/systemd/etc/udev/rules.d/80-udev_fpga.rules:
```diff
@@ -2,4 +2,4 @@
 
 KERNEL=="fpga*", ACTION=="add", OWNER="root", GROUP="users", MODE="660", PROGRAM="/usr/local/share/halcs/scripts/generate-dev-idx.sh %k", SYMLINK+="fpga/%c{1}", RUN+="/usr/bin/systemctl --no-block start halcs-ioc@%c{2}.target halcs-ioc@%c{3}.target"
```
I'd first make a commit removing this file, it's not used by the build system anymore.

guilhermerc replied:
I've fixed and properly parameterized the build system (`80-udev_fpga.rules` was actually being used by it) so it's not installed anymore.

ericonr reviewed (commented):
For the commit message, I'd use some word other than "brick"; IMO that implies it's stuck forever in a bad state, instead of just having to be restarted. Probably something like "locks up" ?

guilhermerc committed:
build_cmake: reload udev rules after installation

Note that, as per udevadm man page:
"Reloading rules and databases does not apply any changes to
already existing devices; the new configuration will only be applied to
new events."

guilhermerc committed:
CMakeLists.txt: don't install halcsd udev rules

We only need halcs_generic_udev udev rules.

guilhermerc committed:
CMakeLists.txt: add missing clause

Altough we had halcsd_INSTALL_UDEV_RULES option, it wasn't
parameterizing udev rules' installation.
